kickstarts/tigeros.ks

```diff
@@ -431,6 +433,8 @@ tigeros-dash-to-dock
 tigeros-ff-profile
 tigeros-logos
 tigeros-release
+tigeros-repkg-jflap
```

Comment:
Remove repkg RPMs, and only install them through the postinstall script. 
These can potentially bring in legal disputes if installed by default